### DISCUSSION TITLE
Exome vs RNAseq

### DISCUSSION THREAD

**Discussion by ajgentles** [milaboratory/mixcr#1074] [Q&A] [answered]
We happen to have (bulk) exome and RNAseq on the same samples and I was curious whether people find one or the other is better for mixcr? We plan to try both and compare, but that's just one dataset.

**Comment by PoslavskySV** [comment, marked as answer]
As individual TCRs/BCRs are represented in RNA level by several (lots) of transcripts in comparison to DNA level (only one rearrangement) there is higher probability to extract/assemble CDR3 region from RNAseq. So RNA-Seq is definitely better.
